src/cloudFoundryConfigClient.ts

```diff
@@ -310,6 +314,8 @@ export async function loadAndRepeat(
           `auto-refreshing config after waiting ${interval} seconds`
         );
         updateFunc(await load(config, params, loadLocalFunc, loadRemoteFunc));
+        console.log('in func', configEvents)

```

Comment:
It might be a good idea to use `console.debug` to record the raising of an event. People would find that useful when they have the `debug` flag turned on.

Reply:
added `console.debug` statements